README.md

```diff
@@ -212,6 +212,6 @@ O1(
 
 # ToDo
 
-* test nullability cases
-* allow printing of `java*` classes
-* fix unicode line breaking with certain characters only under icu4j library
+* Test nullability cases,
```

Comment:
I don't think there should be commas and `and` here. You're a college student correct? Not sure which style guide you are using, but per APA [here](https://blog.apastyle.org/apastyle/2010/03/lists-part-5-bulleted-lists.html) and [here](http://rasmussen.libanswers.com/faq/32609), you only use commas if the list is within a sentence. 

Technically this is a list of topics to come back to so I would rather not have it read like a sentence. Especially because if I delete an item I'd have to reformat other lines, which is usually a no-no in software dev.